website/react-magma-docs/src/pages/api-intro/changelog.mdx

```diff
@@ -5,7 +5,6 @@ order: 6
 
 <PageContent componentName="changelog" type="api_intro">
 
-
```

Comment:
I think this is missing the items for 3.8 and 3.9 https://react-magma.cengage.com/version/3.9.0/api-intro/changelog/